packages/x-components/src/components/base-grid.vue

```diff
@@ -69,8 +70,12 @@
      *
      * @public
      */
-    @Prop({ required: true })
-    protected items!: GridItem[];
+    @Prop({ required: false })
```

Comment:
FYI, by default props are optional, you don't need to add the `required: false`, and it is how this has been done in the rest of the project.

Reply:
I forgot to remove it. My fault.